banks-client/src/lib.rs

```diff
@@ -581,7 +581,9 @@ mod tests {
         // It creates a runtime explicitly (no globals via tokio macros) and calls
         // `runtime.block_on()` just once, to run all the async code.
 
-        let genesis = create_genesis_config(10);
+        // Fund payer account with minimal transaction fee for one signature, plus some extra.
+        let mint_lamports = 10 + solana_sdk::fee::FeeStructure::default().lamports_per_signature;
+        let genesis = create_genesis_config(mint_lamports);
```

Comment:
Previously, `load_accounts()` [read BlockHashQueue for fee_calculator.lamports_per_signature](https://github.com/solana-labs/solana/blob/master/runtime/src/accounts/mod.rs#L74), then use it as "congestion_multiplier". [`fee_calculator`](https://github.com/solana-labs/solana/blob/master/sdk/program/src/fee_calculator.rs#L11-L48) perhaps should not be used anymore. In any case, fee_calculator.lamports_per_signature would be zero in this test, therefore would have set transaction fee to `0`. 

With the change in this PR, eg removing congestion_multiplier all together, this transaction transaction will have fee of 1 signature (eg 5000 lamports). Account this in mint account corrects the test.

Reply:
Bunch of other tests, including those form local-cluster tests, might have been broken for the same reason: 
- genesis is configured with 0 lamports_per_signature in fee_calculator. 
- it is read in as congestion_multipler, that effectively set transaction fee to 0.
- tests were setup with low balance since transaction has zero fee.
- with changes in this PR, transaction will have 5000 lamports per signature fee, that fails transactions with low payer balance. 

One option is to initialize bank.fee_structure from blockhash_queue.fee_calculate.lamport_per_signature, instead of default, that should keep all exist tests success without further changes. But that itself has risk, should be in its own pr.